### PR TITLE
Backend rating value

### PR DESCRIPTION
**Pull request by ghost**
Added logic so that average rating value from the reviews, are sending with a GET-request. If there are 4 reviews or less, average rating-value (imdb-rating) is coming from OMDB-API.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -102,6 +110,11 @@ app.post("/api/movies/review", (request, response) => {
   postRequest(url, jsonData, response);
 });
 
+app.get("/api/movies/rating/:id", async (req, res) => {
+  const data = await averageRating(req.params.id);
+  res.send(data);
+});
+
```
Comment (AnnaLenaO) [inline src/app.js:121]: If you like you can add some catch(error) etc like in the rest of the routes.

File: src/movies.js [hunk 2]
```diff
@@ -47,3 +47,45 @@ export async function getMovieScreenings(cmsAdapter, id) {
...
+  const imdbMovieID = newArray[0].imdbId;
+  const OMDB_rating = await fetch(
+    "https://www.omdbapi.com/?apikey=bf65b015&i=" + imdbMovieID
+  );
+  const OMDB_payload = await OMDB_rating.json();
+
+  if (newArray.length < 5) {
+    return OMDB_payload.imdbRating;
+  } else {
+    return averageValue.toFixed(1);
```
Comment (AnnaLenaO) [inline src/movies.js]: It is easy to read the code in the function averageRating() and understand what it achives. That is nice.

**Review by Weeezzt — APPROVED** [review]
I think the code was easy to read, looks good. I can agree with Anna-Lena that try/catch is something i often use so that i can see what an error that occurs is.

File: src/app.js (hunk 1 above)
Comment (ghost) [reply src/app.js:121]: Added try, catch to route and to function averageRating in movie.js-file

**Review by AnnaLenaO — APPROVED** [review]

**Pull request merged**